src/benchmarks/micro/libraries/System.Threading/Perf.CancellationToken.cs

```diff
@@ -17,6 +17,7 @@ public class Perf_CancellationToken
         public void RegisterAndUnregister_Serial() => _token.Register(() => { }).Dispose();
 
         [Benchmark(OperationsPerInvoke = 1_000_000)]
+        [BenchmarkCategory(Categories.NoWASM)]
```

Comment:
For the multi-threaded tests, we could introduce a `multithreaded` category and use it to exclude the benchmarks when affinitizing to a single core. Sth like:

```cmd
--filter * --category-exclusion-filter multithreaded --affinity X
```

And the full WASM filter would be:

```cmd
--filter * --category-exclusion-filter nowasm multithreaded
```